src/catalog/table_catalog.cpp

```diff
@@ -364,10 +377,12 @@ bool TableCatalog::InsertTable(oid_t table_oid, const std::string &table_name,
   auto val0 = type::ValueFactory::GetIntegerValue(table_oid);
```

Comment:
Variable names should be more meaningful.

Reply:
Agreed. This is very important. Please fix.